internal/app/machined/pkg/controllers/network/link_status.go

```diff
@@ -257,6 +258,23 @@ func (ctrl *LinkStatusController) reconcile(
 				status.BusPath = driverInfo.BusInfo
 			}
 
+			var pciDev *pci.Device
+
+			pciDev, err = pci.SysfsDeviceInfo(driverInfo.BusInfo)
```

Comment:
in the future we might need to lookup more pci devices, does it makes sense to keep this here?